manim/utils/rate_functions.py

```diff
@@ -155,85 +164,85 @@ def result(t):
 # "lingering", different from squish_rate_func's default params
 
 
-def lingering(t):
+def lingering(t: float) -> typing.Callable:  # not sure of return type
```

Comment:
`# not sure of return type`

should be float

Reply:
Does it return float or it returns a function which returns a float? If it a function what should be its parameters.

Reply:
It's a float. squish_rate_func() returns a function, which is then evaluated at t `squish_rate_func(...)(t)`